catalog/citation/admin.py

```diff
@@ -2,10 +2,50 @@
 from django.contrib import admin
 from django.contrib.admin.helpers import ActionForm
 from django import forms
-
+from django.utils.translation import ugettext_lazy as _
 from .models import AuditCommand, Publication, Note, Tag, Author, Sponsor, Platform, Container, ModelDocumentation
 from .search_indexes import PublicationIndex
 
+class StatusListFilter(admin.SimpleListFilter):
+    # Human-readable title which will be displayed in the
+    # right admin sidebar just above the filter options.
+    title = _('status')
+
+    # Parameter for the filter that will be used in the URL query.
+    parameter_name = 'status'
+
+    def lookups(self, request, model_admin):
+        """
+        Returns a list of tuples. The first element in each
+        tuple is the coded value for the option that will
+        appear in the URL query. The second element is the
+        human-readable name for the option that will appear
+        in the right sidebar.
+        """
+        return (
```

Comment:
It would be good to re-use the Publication.Status Choices so we have a single place to change this data. Otherwise if we add a status, we'll have to remember to add it here as well as the Publication class in `citation/models.py`